code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -807,7 +818,6 @@
 
 //ACID
 
-
```

Comment:
Rest of the toxins below this are really not that usefull or a joke-like chem thats used against one race/againts items